src/src/components/DropDownMenu/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import style from "./dropdownmenu.module.css";
+import ProfilePicture from "pages/capabilities/members/profilepicture";
+
+export default function DropDownMenu({
```

Comment:
Maybe change it to a less generic name, sounds like its a DropDownMenu you can use for anything, but in reality it is for invitations

Reply:
Addressed in https://github.com/dfds/selfservice-portal/pull/133/commits/cb4b6bfc44c8d7dff3c4ea8e1c66e2dc416c35bd